README.md

```diff
@@ -375,6 +375,7 @@ These can be useful for osint and social engineering.
 - [Infringement Report](https://infringement.report/) - The web's best image copyright infringement search tool
 - [Tineye](https://tineye.com/) - Image search and recognition company
 - [Flickr](https://flickr.com/search/) - Home to tens of billions of photos and 2 million groups
+- [Sogou](https://pic.sogou.com/) - Chinese technology company that offers a search engine.
```

Comment:
I can't find the proper default description of the service so I'm okay with this one. 
Just remove the ending period since no item has that.

Sorry for the precision :P